packages/validator/src/services/indices.ts

```diff
@@ -110,6 +115,44 @@ export class IndicesService {
         newIndices.push(validatorState.index);
       }
     }
+
+    logValidatorStatuses(allValidators, this.logger);
+
     return newIndices;
   }
 }
+
+function logValidatorStatuses(allValidators: Map<ValidatorStatus, ValidatorResponse[]>, logger: Logger): void {
+  const loggedValidatorStates: {[key: string]: ValidatorStatus[]} = {
+    "Validator activated": ["active"],
+    "Validator pending": ["pending_initialized", "pending_queued"],
+    "Validator withdrawn": ["withdrawal_done"],
+    "Validator exited": ["exit_slashed", "exit_unslashed"],
+  };
+
+  const logFormatter = (message: string, validatorState: ValidatorResponse): void => {
+    logger.info(message, {
+      pubkey: toHexString(validatorState.validator.pubkey),
+      index: validatorState.index,
+    });
+  };
+  Object.keys(loggedValidatorStates).map((key) => {
+    getValidatorStatesByStatuses(loggedValidatorStates[key], allValidators).forEach(
+      (validatorState: ValidatorResponse) => {
+        logFormatter(key, validatorState);
+      }
+    );
+  });
+
+  logger.info("Initialized validators", {
+    total: allValidators.size,
+    active: allValidators.get("active")?.length,
```

Comment:
what do we want to log here if there are no `active` validators? I think it should be the number `0`

```suggestion
    active: allValidators.get("active")?.length ?? 0,
```